### ISSUE TITLE
Smooth Scroll to Sections

### ISSUE THREAD

**Issue by mbbischoff** [nytimes/Open-Source-Science-Fair#1]
Would it be easy / possible to scroll smoothly to the section that you clicked when you click section from the header?

**Comment by lggarrison** [comment]
Yes. We can add this kind of feature. We just need to add jQuery to the page. And some code similar to this.

http://css-tricks.com/snippets/jquery/smooth-scrolling/

I have some work that has priority right now. I'll try to squeeze this in sometime this week.
